build_package.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+VERSION=$(<VERSION)
```

Comment:
I think we should pull this with 'git describe' because it's better able to handle intermediate commits. The problem with a VERSION file is that it doesn't really handle additional commits and so bad things can sneak in.

Reply:
Done